src/components/BatchSearchForm.vue

```diff
@@ -4,21 +4,25 @@
       <h5 v-if="!hideTitle" class="py-2 h6 text-uppercase text-muted">
         {{ $t('batchSearch.heading') }}
       </h5>
-      <div class="card w-100" :class="{ 'border-0': hideBorder }">
+      <div :class="{ 'border-0': hideBorder }" class="card w-100">
         <div class="card-body pb-1">
-          <b-form-group label-size="sm" :label="`${$t('batchSearch.name')} *`">
-            <b-form-input v-model="name" type="text" required></b-form-input>
+          <b-form-group :label="`${$t('batchSearch.name')} *`" class="batch-search-form__name" label-size="sm">
+            <b-form-input v-model="name" required type="text"></b-form-input>
           </b-form-group>
-          <b-form-group label-size="sm" class="mb-0" :label="`${$t('batchSearch.fileLabel')} *`">
+          <b-form-group
+            :label="`${$t('batchSearch.fileLabel')} *`"
+            class="batch-search-form__fileLabel mb-0"
+            label-size="sm"
+          >
             <template slot="description">
               <div v-html="$t('batchSearch.fileDescription')"></div>
             </template>
             <b-form-file
               v-model="csvFile"
               :placeholder="$t('batchSearch.filePlaceholder')"
+              :state="Boolean(csvFile)"
```

Comment:
I know it was already there but this state should be a computed property. I would have preferred the shorthand `!!` too.